src/server.rs

```diff
@@ -145,6 +147,128 @@ impl std::fmt::Display for CleanupState {
     }
 }
 
+static INIT: Once = Once::new();
+static TRACKED_PARAMETERS: Lazy<HashSet<String>> = Lazy::new(|| {
+    INIT.call_once(|| {
```

Comment:
This is a strange way to log this. Also, why are you logging this?

Reply:
removed log line, co-pilot auto-filled it haha